contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -107,33 +108,24 @@ var EditMetadataView = BaseViews.BaseEditableListView.extend({
     this.edit_list = new EditMetadataList({
       collection:this.collection,
       new_content : this.new_content,
-      upload_files : this.upload_files,
+      new_exercise : this.new_exercise,
+      new_topic: this.new_topic,
       el: this.$("#topic_tree_selector"),
       model: this.model,
       container: this
     });
     this.edit_list.handle_if_individual();
   },
-  load_preview:function(){
-     this.preview_view = new Previewer.PreviewView({
-      modal:false,
-      el: this.$("#metadata_preview"),
-      model:null
-    });
+  load_preview:function(view){
```

Comment:
Localize preview and questions tabs to items, not container (simplifies recording changes)